guides/Front-end-strategy.md

```diff
@@ -0,0 +1,54 @@
+# Front-end strategy
+
+### What we use
+
+#### Foundation 6 framework
+
+Use only required components and libraries. We don't need to use normalize with Foundation 6, it included in foundation-global-styles mixin.
+
+#### Grunt
+
+Use grunt tasks for creating svg sprites. You can use `grunt` command to run grunt tasks.
+
+#### Autoprefixer
+
+Use Autoprefixer gem or Autoprefixer of [grunt-postcss](https://github.com/nDmitry/grunt-postcss) task in Shopify projects.
+
+#### Bower
+
+Use bower with bower installer to manage libraries and control their versions. [Using front-end libraries](https://github.com/DVELP/cookbook/blob/front-end-strategy/guides/Using%20front-end%20libraries.md)
+
+#### Front-man
+
+[Front-man](https://github.com/DVELP/front_man) It includes utilities classes, z-index variables and helpers for sprite and image tags.
```

Comment:
The rendered document doesn't look good. Let's inline those links so that text reads niceer?

![image](https://cloud.githubusercontent.com/assets/7405/19303276/b5d9774c-9070-11e6-887b-a4fb267c47cb.png)